helm/checksum_annotations.md

```diff
@@ -0,0 +1,43 @@
+# Helm Checksum Annotations
+
+## Overview
+
+- We add checksum annotations to our deployments and daemonsets to trigger
+fresh pods when their configmaps or secrets have changed.
+- This reduces how often we need to roll pods which can be disruptive.
+- Changes to other resources like PSP and RBAC are applied without needing to
```

Comment:
From the discussion earlier https://gigantic.slack.com/archives/C3C7ZQXC1/p1600771250010300
we think we only need annotations for CMs and secrets.

If there is a case we're missing please chime in.